Packages/Application/Neos.NeosIo.Event/Resources/Private/Templates/Page/Partials/SiteHeader.html

```diff
@@ -0,0 +1,48 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+{namespace media=TYPO3\Media\ViewHelpers}
+
```

Comment:
Can we reuse or is it different from Neos.io?

To me (at first sight) it looks as if just some CSS classes are different; if this was the case we could change them via Fusion :)